resources/dicts/thoughts/other.json

```diff
@@ -213,7 +213,7 @@
             "Hisses at a missed catch",
             "Is hoping they can find something to fill their belly tonight",
             "Licks at an old, aching scar in an attempt to soothe it",
-            "Was almost found by a c_n patrol",
+            "Was almost found by an c_n patrol",
```

Comment:
this should actually always be 'a' because of our little system that will change it to 'an' if the clan name starts with a vowel